### PR TITLE
Test/homepage and errors

### PR DESCRIPTION
**Pull request by tialaaa**
# PR Template

### Types of changes made?

- [ ]  documentation
- [x]  testing
- [ ]  functionality
- [ ]  styling
- [x]  refactor

### What does this PR do?

Added Cypress environment variable for API keys to use in fetch calls.
Added fixture data for request stubbing.

Writes test for all Homepage functionality, including checks for:
- Header content
- Temporary loading message
- Page display after data loading is complete
- Interactive cards and their affect of colors and displayed messages

Writes test for error handling - both server-side and client-side.

Also renames the cleaning function and env variables for more clarity.

### CODE REVIEW

File: cypress/e2e/error-handling.cy.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    cy.intercept({
+      pathname: '/api/v1/parks',
+      query: {
+        limit: '469',
+        api_key: npsApiKey,
+      },
+    }, {
+      statusCode: 500,
+      body: {
+        message: 'Internal Server Error'
```
Comment (CapCinematic) [inline cypress/e2e/error-handling.cy.js:14]: Checking for proper status codes, this looks great and a clear message of what could be going on.

File: cypress/e2e/error-handling.cy.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        api_key: npsApiKey,
+      },
+    }, {
+      statusCode: 200,
+      fixture: 'parks'
+    })
+    cy.visit('http://localhost:3000/acadia')
+
+    cy.get('.logo-link').should('include.text', 'ParkPassport')
+      .get('.logo').should('have.attr', 'src', '/PP-logo.png')
```
Comment (CapCinematic) [inline cypress/e2e/error-handling.cy.js:38]: Very thoroughly checking your links awesome!

**Review by CapCinematic — APPROVED** [review]
Over all these tests are very thorough and considerate of the user experience. From my knowledge, everything seems accounted for.

**Comment by tialaaa** [comment]
Thanks for reviewing @CapCinematic !

**Pull request merged**